lib/improver/cli/standardise.py

```diff
@@ -87,20 +87,20 @@ def process(source_data: cli.inputcube,
             the source data is not a MaskedArray
             nanmask - if the source data is a MaskedArray extrapolated points
             will be masked; otherwise they will be set to NaN
-        landmask_vicinity (float):
+        land_sea_mask_vicinity (float):
             Radius of vicinity to search for a coastline, in metres.
-        regridded_title (str or None):
+        regridded_title (str):
             New "title" attribute to be set if the field is being regridded
             (since "title" may contain grid information). If None, a default
             value is used.
-        attributes_dict (dict or None):
+        new_metadata (dict):
```

Comment:
Illustration of why naming this "attribute_change" or similar is sensible, since other metadata are updated through different arguments.